hudi-utilities/src/test/java/org/apache/hudi/utilities/sources/TestGcsEventsHoodieIncrSource.java

```diff
@@ -215,7 +215,7 @@ private HoodieRecord getGcsMetadataRecord(String commitTime, String filename, St
 
   private HoodieWriteConfig getWriteConfig() {
     return getConfigBuilder(basePath(), metaClient)
-            .withArchivalConfig(HoodieArchivalConfig.newBuilder().archiveCommitsWith(2, 3).build())
+        .withArchivalConfig(HoodieArchivalConfig.newBuilder().archiveCommitsWith(4, 5).build())
             .withCleanConfig(HoodieCleanConfig.newBuilder().retainCommits(1).build())
```

Comment:
Not sure what the purpose of this change?

Reply:
We cannot set min archive commits to 2 anymore because the metadata table's number of retained commits for cleaning is 3 and the metadata table's min archive commits is the same as data table, so the min archive commits have to be larger than 3 (number of retained commits for cleaning) now.  That's why all of the relevant tests have to be changed.

Reply:
Should we also tweak the min cleaning commits for MDT then. It seems there is conflict for the data table archiving and MDT cleaning commits here.

Reply:
There is no conflict in the production code, where 20 and 30 are set for min and max commits for archival.  And there's a reason to keep a few versions for MDT from cleaning.  It's only in some tests that we take shortcuts.